app/containers/LoginServices.tsx

```diff
@@ -271,77 +264,51 @@ class LoginServices extends React.PureComponent<ILoginServicesProps, ILoginServi
 		return Base64.encodeURI(JSON.stringify(obj));
 	};
 
-	openOAuth = ({ url, ssoToken, authType = 'oauth' }: IOpenOAuth) => {
-		const { navigation } = this.props;
+	const openOAuth = ({ url, ssoToken, authType = 'oauth' }: IOpenOAuth) => {
 		navigation.navigate('AuthenticationWebView', { url, authType, ssoToken });
 	};
 
-	transitionServicesTo = (height: number) => {
-		const { servicesHeight } = this.state;
-		if (this._animation) {
-			this._animation.stop();
-		}
-		this._animation = Animated.timing(servicesHeight, {
-			toValue: height,
-			duration: 300,
-			easing: Easing.inOut(Easing.quad),
-			useNativeDriver: false
-		}).start();
-	};
-
-	toggleServices = () => {
-		const { collapsed } = this.state;
-		const { services } = this.props;
-		const { length } = Object.values(services);
-		if (collapsed) {
-			this.transitionServicesTo(SERVICE_HEIGHT * length);
-		} else {
-			this.transitionServicesTo(SERVICES_COLLAPSED_HEIGHT);
-		}
-		this.setState((prevState: ILoginServicesState) => ({ collapsed: !prevState.collapsed }));
-	};
-
-	getSocialOauthProvider = (name: string) => {
+	const getSocialOauthProvider = (name: string) => {
 		const oauthProviders: IOauthProvider = {
-			facebook: this.onPressFacebook,
-			github: this.onPressGithub,
-			gitlab: this.onPressGitlab,
-			google: this.onPressGoogle,
-			linkedin: this.onPressLinkedin,
-			'meteor-developer': this.onPressMeteor,
-			twitter: this.onPressTwitter,
-			wordpress: this.onPressWordpress
+			facebook: onPressFacebook,
+			github: onPressGithub,
+			gitlab: onPressGitlab,
+			google: onPressGoogle,
+			linkedin: onPressLinkedin,
+			'meteor-developer': onPressMeteor,
+			twitter: onPressTwitter,
+			wordpress: onPressWordpress
 		};
 		return oauthProviders[name];
 	};
 
-	renderServicesSeparator = () => {
-		const { collapsed } = this.state;
-		const { services, separator, theme } = this.props;
+	const renderServicesSeparator = () => {
```

Comment:
You don't want to use `renderSomething`.
That's ok for class components, but not for function components, as it's going to get recreated on every render.
The ideal fix is to extract the logic to its own component `<ServicesSeparator />` and send the necessary props.

A _not-so-valid_ fix would be to use `useCallback`, but it's hard to maintain and it's kind of anti-pattern.
There are use cases for this, but 99% of times a separate component is the right way.